.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
       script:
         - make lint
         # Lint the first commit in the PR.
-        - git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx core-validate-commit --no-validate-metadata
+        - \[ -z "$TRAVIS_COMMIT_RANGE" \] || git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx core-validate-commit --no-validate-metadata
```

Comment:
I think there is no need to escape `[` and `]`.

Reply:
`[` and `]` have special meaning in YAML, so I think you do need to escape at least the opening one, but I'm not sure, so let's try...

Reply:
https://github.com/nodejs/node/pull/23741 to test

Reply:
Looks like the result there is malformed YAML and Travis doesn't kick off. Confirmed with https://codebeautify.org/yaml-validator that escaping is needed.

Reply:
You are right, I was tricked by https://github.com/primus/eventemitter3/blob/master/.travis.yml#L20 and many other similar cases but that works because the whole string is wrapped in `''`.